src/leap/bitmask/gui/mainwindow.py

```diff
@@ -1207,9 +1207,13 @@ def _on_user_logged_in(self):
         self._disconnect_login_wait()
 
         user = self._login_widget.get_logged_user()
-        domain = self._providers.get_selected_provider()
-        full_user_id = make_address(user, domain)
-        self._mail_conductor.userid = full_user_id
+        # XXX the widget now gives us the full user id.
+        # this is confusing.
+        #domain = self._providers.get_selected_provider()
+        #full_user_id = make_address(user, domain)
```

Comment:
I expect the backend to fix this regression and go back to the older code. I think the full_user_id denomination is clearer than the current behaviour, which I find confusing.